assets/javascripts/feature.js

```diff
@@ -1,79 +1,118 @@
-// jshint esversion: 6
+function postFeature(latestFeature) {
+  $.ajax({
+    url: '/api/v1/feature',
+    method: 'POST',
+    data: {version: $('#dont-notify').is(':checked') ? 0 : latestFeature}
```

Comment:
Just for the record: As mentioned in chat, `$('#dont-notify').is(':checked')` does not work because at the time this function runs `$('#dont-notify')` is not part of the DOM anymore.